kzz7yh-f04172/cod-ve09v2_kzz7yh-f04172.xml

```diff
@@ -226,11 +227,12 @@
             </div>
             <div xml:id="kzz7yh-f04172-Dd1e394">
                <head xml:id="kzz7yh-f04172-Hd1e396">Quaestio 2</head>
+               <head type="question-title">Utrum plures creature immediate tantum processerit ab ipso et videtur quod non</head>
```

Comment:
In our normalized headings we're going to expand "e" to "ae" when appropriate. 

Here the word should be plural "creatures"

```suggestion
               <head type="question-title">Utrum plures creaturae immediate tantum processerit ab ipso et videtur quod non</head>
```